guides/installation/setups/index.md

```diff
@@ -0,0 +1,44 @@
+---
+nav:
+  title: Setups
+  position: 3000
+
+---
+
+# Setups
+
+## Set up your own environment
+
+Head over to the [Requirements](../requirements) section to install and configure the necessary services like a database and a webserver to a Unix system like Linux, macOS, WSL etc.
+
+## Development setup
+
+::: info
+Technically there is no real difference between a Development Setup and a Production Setup, they only differ on performance and security optimizations.
+:::
+
+> \* These setups are provided and officially supported by shopware AG.
+
+* [Docker*](docker) - This is a Docker setup for Shopware 6. It is a lightweight and easy way to get started with Shopware. It uses Docker Compose to manage the services and is suitable for local development.
+
+* [Symfony CLI*](symfony-cli) - This setup uses the Symfony CLI to run Shopware. It is the default way to run Symfony applications and is also suitable for Shopware.
+
+* [Devenv*](devenv) - This is a setup that manages all necessary services. A description file in the source code manages the versions of these services. This setup works for Linux, WSL and macOS.
+
+* [Dockware](https://dockware.io/getstarted) - This is a managed docker setup for Shopware 6 by Shopware agency [dasistweb](https://www.dasistweb.de/).
+
+* [DDEV](https://notebook.vanwittlaer.de/ddev-for-shopware/less-than-5-minutes-install-with-ddev-and-symfony-flex) - Docker-based PHP development environments, works on all platforms and is generic enough to be used for any PHP project. [Project Page](https://ddev.com/)
```

Comment:
I would remove the note about the officially supported by Shopware AG and, after that, split it up into two sections below Development setup ... Section 1: Officially supported by Shopware AG ... Section 2: Community supported and then have the lists and remove the * behind docker, Symfony CLI, even...